va/caa.go

```diff
@@ -196,11 +227,36 @@ func (va *ValidationAuthorityImpl) validateCAASet(caaSet *CAASet) (present, vali
 		return true, true
 	}
 
+	// Per RFC 6844 Section 5.3 "issueWild properties MUST be ignored when
+	// processing a request for a domain that is not a wildcard domain". Skip
+	// checking `caaSet.Issuewild` if `wildcard` is false.
+	if wildcard && len(caaSet.Issuewild) > 0 {
+		// We are processing CAA for a wildcard authorization identifier and there
+		// was at least one Issuewild CAA record in the set. This means we must
+		// check that our CAA identity is present in the issueWild set.
+		//
+		for _, caaIssuewild := range caaSet.Issuewild {
+			if extractIssuerDomain(caaIssuewild) == va.issuerDomain {
+				va.stats.Inc("CAA.IssuewildAuthorized", 1)
+				return true, true
+			}
+			// Since our identity is *not* present we must without checking the
+			// `caaSet.Issue` identities because per RFC 6844 Section 5.3 if the domain
+			// is a wildcard domain and there is at least one issuewild property then
+			// "all issue properties MUST be ignored"
+			return true, false
```

Comment:
Oh wait, yeah shouldn't this be moved out a level?

Reply:
Yup - good catch! I think this would have rejected issuance in some cases that should be allowed where the first issuewild in the CAA set that doesn't match would abort further processing. I'll fix - thanks!

Reply:
Fixed in b10b7fd